test/controller.test.ts

```diff
@@ -3456,23 +3547,12 @@ contract('Controller', ([owner, accountOwner1, accountOwner2, accountOperator1,
       // past time after expiry
       await time.increase(60 * 61) // increase time with one hour in seconds
       // set price in Oracle Mock, 150$ at expiry, expire ITM
-      await oracle.setExpiryPrice(
-        await shortOtoken.underlyingAsset(),
-        new BigNumber(await shortOtoken.expiryTimestamp()),
-        new BigNumber(150).times(new BigNumber(10).exponentiatedBy(18)),
-      )
-      await oracle.setIsFinalized(
-        await shortOtoken.underlyingAsset(),
-        new BigNumber(await shortOtoken.expiryTimestamp()),
-        true,
-      )
-      await oracle.setIsDisputePeriodOver(
-        await shortOtoken.underlyingAsset(),
-        new BigNumber(await shortOtoken.expiryTimestamp()),
-        true,
-      )
+      const expiry = new BigNumber(await shortOtoken.expiryTimestamp())
+      await oracle.setExpiryPriceFinalizedAllPeiodOver(weth.address, expiry, new BigNumber(150).times(1e18), true)
+      await oracle.setExpiryPriceFinalizedAllPeiodOver(usdc.address, expiry, new BigNumber(1).times(1e18), true)
```

Comment:
I think it is better to not use this function as it doesn't not exist in Oracle module, I understand that this mock function is mimicking the same behavior, but better to follow same flow IMO.